gamewin.h

```diff
@@ -574,15 +574,15 @@ class Game_window {
 	void read_saveinfo(IDataSource *in,
 	                   SaveGame_Details *&details,
 	                   SaveGame_Party  *&party);
-#ifdef HAVE_ZIP_SUPPORT
 private:
+#ifdef HAVE_ZIP_SUPPORT
 	bool get_saveinfo_zip(const char *fname, char *&name,
 	                      Shape_file *&map,
 	                      SaveGame_Details *&details,
 	                      SaveGame_Party  *&party);
+#endif
 	void restore_flex_files(IDataSource &ds, const char *basepath);
```

Comment:
I think you meant "move *restore_flex_files* outside of HAVE_ZIP_SUPPORT" in the commit message.